packages/graphql-testing/src/MockGraphqlClient.ts

```diff
@@ -0,0 +1,24 @@
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
+
+import Operations from './Operations';
+
+import MemoryApolloLink from './MemoryApolloLink';
+
+export default class MockGraphQLClient extends ApolloClient<unknown> {
+  readonly graphQLRequests: Operations;
+  private memoryLink: MemoryApolloLink;
+
+  constructor(options: ApolloClientOptions<unknown>) {
+    const memoryLink = new MemoryApolloLink();
+    super({
+      ...options,
+      link: memoryLink.concat(options.link),
+    });
+    this.memoryLink = memoryLink;
+    this.graphQLRequests = memoryLink.requests;
```

Comment:
IMO this is still a bit awkward, can this instance property also move to just be on the client?